### DISCUSSION TITLE
Issues related to interpreting result

### DISCUSSION THREAD

**Discussion by shengkuang2** [hahnlab/CAFE5#197] [Q&A] [answered]
My materials are nine allotetraploids, which are divided into two subgenomes, A and B, as shown in the figure. It can be observed that the bases of both materials A and B are mostly expanded with minimal contraction. How should this phenomenon be explained?
![未标题-1](https://github.com/user-attachments/assets/676b96e0-6edf-4faa-b035-9ef7b077f0b0)

**Comment by benfulton** [comment, marked as answer]
Dr. Hahn thinks it’s just that you don’t have any species outside these, so reciprocal losses after WGD show up as gains on the other side of the root.
